src/posts/features-and-projects.md

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Feature Updates and Other Projects'
+date: '2022-03-09'
+authors:
+  - 'BenW'
+  - 'Cdr_Maverick'
+  - 'Kevin'
+  - 'Sven [de en]'
+  - 'Valastiri'
+category: 'UPDATE'
+metaImage: '/img/notam-images/march2022/dcdu-dep-req.png'
+metaAlt: 'FlyByWire Simulations'
+---
+
+## Overview
+
+We hope that you have enjoyed the series of in-depth NOTAMs we published in February. As the team at FlyByWire Simulations continues to release new features and fine-tune those we already have, we never stop thinking of new ways to improve the experience flying the A32NX.
+
+Keeping up with the many features we release each day onto our Development branch can be a little daunting. With that in mind this NOTAM calls to attention previously released features and updates as well as a sneak peek into flyPadOS v3! Additionally, we shine a light on the state of a couple of our other FlyByWire projects - Discord Bot and Documentation.
+
+---
+
+## Feature Updates
+
+### Display Typeface Development
+
+We believe in a holistic approach to aircraft development - we do not sacrifice one aspect of the aircraft for another, and this means we put as much attention into visuals as we do into systems. As part of this quest for visual accuracy, we have spent countless hours creating custom typefaces that replicate the look of Airbus’ many display fonts. At last count, there are five cockpit display typefaces on both the A320 and A380 - the EIS font, the ISIS font, the FMS font (A320 only), the DCDU font (A320 only) and the RMP font (A380 only). This does not include the seven-segment displays dotting the cockpit panels as well as the A380 EFIS control panel LCDs.
+
+> How do we do them, and where do we start?
+
+First, we determine whether the font is anti-aliased. If it is, we look for head-on closeups of the glyphs (letters, numbers and symbols) and carefully trace each glyph. If not, we look for underexposed display shots which allow us to discern every pixel on the glyphs, then trace the outlines of each glyph (the ‘path’) to create a pixel-perfect font. Once we are done with this, we determine the amount of spacing between letters, which in this case is consistent for all characters within each of the fonts since all of aforementioned fonts are monospaced.
```

Comment:
```suggestion
First, we determine whether the font is anti-aliased. If it is, we look for head-on closeups of the glyphs (letters, numbers, and symbols) and carefully trace each glyph. If not, we look for underexposed display shots that allow us to discern every pixel on the glyphs, then trace the outlines of each glyph (the ‘path’) to create a pixel-perfect font. Once we are done with this, we determine the amount of spacing between letters, which in this case, is consistent for all characters within each of the fonts since all of the fonts mentioned above are monospaced.
```